src/main/main.cpp

```diff
@@ -262,12 +262,9 @@ checkInitialized(Application::pointer app)
 }
 
 static int
-catchup(Config const& cfg, uint32_t to, uint32_t count,
+catchup(Application::pointer app, uint32_t to, uint32_t count,
         Json::Value& catchupInfo)
 {
-    VirtualClock clock(VirtualClock::REAL_TIME);
```

Comment:
why not do `VirtualClock& clock = app->getClock()`, this avoids having all those `app->getClock()` calls